common/app/views/fragments/newHeader.scala.html

```diff
@@ -17,10 +17,6 @@ <h1 class="u-h">The Guardian</h1>
         <nav class="new-header__nav">
             <a href="#main-menu" class="new-header__nav__menu-button js-change-link" tabindex="0">
                 <div class="new-header__burger-icon js-animate-menu">
```

Comment:
Should this still be a div? Our current nav has `<i>`:
![image](https://cloud.githubusercontent.com/assets/8774970/17326006/925971a2-58a6-11e6-9c11-f343bc4e5609.png)